twitter.css

```diff
@@ -0,0 +1,147 @@
+body {
```

Comment:
Good Job. Be there is a lot of more work to be done... You did good with the half screen of blue and the buttons . However there are no input fields.. doesn't have a hover effect nor a footer. Take some time to come back to this to complete . CSS takes a lot of patience. but good practice helps a lot !